motionplan/motionPlanner_test.go

```diff
@@ -515,57 +504,14 @@ func TestReachOverArm(t *testing.T) {
 
 	// the plan should no longer be able to interpolate, but it should still be able to get there
 	opts = map[string]interface{}{"max_ik_solutions": 100, "timeout": 150.0}
-	plan, err = PlanMotion(context.Background(), logger.Sugar(), goal, xarm, frame.StartPositions(fs), fs, nil, nil, opts)
-	test.That(t, err, test.ShouldBeNil)
-	test.That(t, len(plan), test.ShouldBeGreaterThan, 2)
-}
-
-func TestPlanMapMotion(t *testing.T) {
```

Comment:
why are we deleting this test?

Reply:
Its redundant as it is covered by other tests in the motion package.  @biotinker do you have thoughts here?

Reply:
I'd prefer to keep this. It's currently fully overlapping with other tests we have, especially in kinematicsbase and in the builtin motion service, but once we swap diff drive to use PTGs, this will be more important in making sure we don't have regressions in our ability to plan for a `frame.New2DMobileModelFrame`